packages/styling/src/interfaces/ITheme.ts

```diff
@@ -4,6 +4,8 @@ import { ISemanticColors } from './ISemanticColors';
 import { ITypography, IPartialTypography } from './ITypography';
 import { ISpacing } from './ISpacing';
 
+export type IScheme = 'default' | 'neutral' | 'soft' | 'strong';
```

Comment:
should we make this an enum (like VariantThemeType in the variants project)?

Reply:
In general we should be avoiding the use of enums. We need to incorporate this into our standards but for now see #6386